src/analysis/complex/upper_half_plane/basic.lean

```diff
@@ -8,7 +8,7 @@ import linear_algebra.special_linear_group
 import analysis.complex.basic
 import group_theory.group_action.defs
 import linear_algebra.general_linear_group
-
+import geometry.manifold.charted_space
```

Comment:
I don't think you need this import, just the import of the file where `topological_space.opens` is defined.